bycycle/features/features.py

```diff
@@ -0,0 +1,146 @@
+"""Quantify the shape of oscillatory waveforms on a cycle-by-cycle basis."""
+
+import pandas as pd
+
+from bycycle.features.shape import compute_shape_features
+from bycycle.features.burst import compute_burst_features
+from bycycle.burst import detect_bursts_cycles, detect_bursts_df_amp
+
+###################################################################################################
+###################################################################################################
+
+def compute_features(sig, fs, f_range, center_extrema='peak', burst_detection_method='cycles',
```

Comment:
for brevity, I think `burst_detection_method` could probably be `burst_method`. 
Likewise, and if so, then `burst_detection_kwargs` could probably be `burst_kwargs`.

Reply:
Renamed as you suggested (a5f4ed8).